spm_funcs.py

```diff
@@ -50,12 +51,23 @@ def get_spm_globals(fname):
         Filename of file containing 4D image
 
     Returns
+
```

Comment:
The [Numpy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html) wants the underline to go directly under the heading here - so no extra blank line.